### PR TITLE
Docker Privileged Mode Support

### PR DESCRIPTION
**Pull request by feuer93**
Based on https://github.com/jenkinsci/mesos-plugin/pull/69 I've implemented a mergeable Version for the support of privileged mode for docker containers.

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/mesos/MesosSlaveInfo.java [hunk 1]
```diff
@@ -231,6 +236,10 @@ public String getNetworking() {
     public boolean hasPortMappings() {
       return portMappings != null && !portMappings.isEmpty();
     }
+
+    public Boolean getDockerPrivilegedMode() {
+      return dockerPrivilegedMode != null ? dockerPrivilegedMode : false;
```
Comment (vinodkone) [inline src/main/java/org/jenkinsci/plugins/mesos/MesosSlaveInfo.java]: why not just return dockerPrivilegedMode directly?

File: src/main/java/org/jenkinsci/plugins/mesos/MesosCloud.java [hunk 2]
```diff
@@ -470,7 +470,7 @@ public boolean configure(StaplerRequest request, JSONObject object)
 
               containerInfo = new MesosSlaveInfo.ContainerInfo(
                   containerInfoJson.getString("type"),
-                  containerInfoJson.getString("dockerImage"), volumes, parameters, networking, portMappings);
+                  containerInfoJson.getString("dockerImage"), volumes, parameters, networking, portMappings, containerInfoJson.getBoolean("dockerPrivilegedMode"));
```
Comment (vinodkone) [inline src/main/java/org/jenkinsci/plugins/mesos/MesosCloud.java]: can you make the privileged mode as argument 3 to be next to "type" and "dockerImage"?

also, not yours, but mind putting each argument on its own line?

**Comment by vinodkone** [comment]
Looks good. Some minor comments and I'll get this committed. Thanks!

File: src/main/java/org/jenkinsci/plugins/mesos/MesosSlaveInfo.java (hunk 1 above)
Comment (feuer93) [inline src/main/java/org/jenkinsci/plugins/mesos/MesosSlaveInfo.java]: I Thougth this should be the default setting for docker containers. Nevertheless, a default setting will be set on config save, I'll change that

**Comment by feuer93** [comment]
changes commited!

**Pull request merged**